.coderabbit.yaml

```
@@ -28,7 +27,7 @@ reviews:
     issue_assessment:
       mode: warning
   auto_review:
-    base_branches: ["develop", "release/.*"]
+    base_branches: ["develop", "release/.*", "epic/.*"]
```

Let's keep these alphabatized
```suggestion
    base_branches: ["develop", "epic/.*", "release/.*"]
```